package.json

```diff
@@ -187,7 +187,7 @@
     "simplex-noise": "^3.0.1",
     "stream-browserify": "^3.0.0",
     "stream-http": "^3.2.0",
-    "styled-components": "^5.3.6",
+    "styled-components": "^6.0.7",
```

Comment:
Not purely chakra related, but ensures we get the latest emotion minor.
This is only really used by the wherever plugin

```shell
❯ npm why styled-components
styled-components@6.0.7
node_modules/styled-components
  peer styled-components@"*" from @styled-icons/octicons@10.47.0
  node_modules/@styled-icons/octicons
    @styled-icons/octicons@"^10.47.0" from @wherever/react-notification-feed@0.4.4
    node_modules/@wherever/react-notification-feed
      @wherever/react-notification-feed@"^0.4.4" from the root project
  peer styled-components@">=5" from @wherever/react-notification-feed@0.4.4
  node_modules/@wherever/react-notification-feed
    @wherever/react-notification-feed@"^0.4.4" from the root project
  styled-components@"^5.3.6" from the root project
```

<img width="575" alt="image" src="https://github.com/shapeshift/web/assets/17035424/2cac5f6d-0791-4b3b-9b2f-cdc1ea8ad859">